ExampleCode/C++/API/API.cpp

```diff
@@ -0,0 +1,916 @@
+#include <iostream>
+#include <string>                    // std::string, std::to_string;

```

Comment:
Don't need the `// std::string ...` comment anymore since you've already added them below.